MoveNet_Vs_PoseNet.ipynb

```diff
@@ -319,12 +321,7 @@
     "    keypoints = parse_output(template_heatmaps, template_offsets, 0.3)\n",
     "\n",
     "    output_img, scoreList = draw_kps(keypoints, template_image)\n",
-    "    \n",
-    "#     for key, value in scoreList.items():\n",
-    "#         scoreList[key] = sigmoid(value)\n",
     "        \n",
-    "    scoreList = {key:sigmoid(value) for key, value in scoreList.items()}\n",
```

Comment:
@bhavikapanara why was this calculation removed and replaced with hardcoded values? I expected to see a bug fix that resolves the error in the score calculation.